spark/src/main/scala/org/apache/spark/sql/delta/actions/actions.scala

```diff
@@ -139,10 +139,15 @@ case class Protocol private (
   extends Action
   with AbstractProtocol
   with TableFeatureSupport {
-  // Correctness check
-  // Reader and writer versions must match the status of reader and writer features
+  // Correctness check.
+  // Reader and writer versions must match the status of reader and writer features.
+  // For the reader features requirement we exclude the column mapping reader version (2)
```

Comment:
This comment describes the case when the reader version is 3, but the reader features are not defined. The code below on the other hand handles the case in which the reader version is 2.